dts/bindings/interrupt-controller/mediatek,adsp_intc.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2023 The ChromiumOS Authors
+# SPDX-License-Identifier: Apache-2.0
+
+description: MediaTek MT8xxx Audio DSP Interrupt Controller
+
+compatible: "mediatek,adsp_intc"
+
+include: [interrupt-controller.yaml, base.yaml]
+
+properties:
+  reg:
+    required: true
+
+  interrupts:
+    required: true
+
+  "#interrupt-cells":
+    const: 3
+
+  status_reg:
```

Comment:
should be `status-reg`, see https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#naming-conventions